Makefile.dep

```diff
@@ -218,6 +218,12 @@ ifneq (,$(filter ng_pktdump,$(USEMODULE)))
   USEMODULE += od
 endif
 
+ifneq (,$(filter ng_pktbuf, $(USEMODULE)))
+  ifeq (,$(filter ng_pktbuf_dynamic, $(USEMODULE)))
```

Comment:
I think we can make this more general: `ifeq(,$(filter ng_pktbuf_%, $(USEMODULE)))`?!